ion-schema-tests-runner/src/generator.rs

```diff
@@ -321,7 +321,7 @@ fn generate_preamble(root_dir_path: &Path) -> TokenStream {
         /// Asserts that a value is or is not valid for a given ISL type.
         fn __assert_value_validity_for_type(value_ion: &str, schema_id: &str, type_id: &str, expect_valid: bool) -> Result<(), String> {
             let schema = __new_schema_system().load_schema(schema_id).unwrap();
-            let isl_type = schema.get_type(type_id).unwrap();
+            let isl_type = schema.get_local_or_imported_type(type_id).unwrap();
```

Comment:
Is it _necessary_ to change this name? Is the benefit worth it for the churn that users would have to deal with?

Reply:
I think we can just keep the name same for now(i.e. `get_type`). The method will also return imported type now, which seems to be reasonable. User can rely on `get_built_in_or_defined_type` for previous behavior.